dbos/system_database.go

```diff
@@ -288,7 +293,7 @@ func (s *sysDB) shutdown(ctx context.Context, timeout time.Duration) {
 	// Allow pgx health checks to complete
 	// https://github.com/jackc/pgx/blob/15bca4a4e14e0049777c1245dba4c16300fe4fd0/pgxpool/pool.go#L417
 	// These trigger go-leak alerts
-	time.Sleep(_DB_CONNECTION_RETRY_DELAY)
+	time.Sleep(backoffWithJitter(0))
```

Comment:
We can actually hard code this sleep to be 500 ms. It is totally unrelated to DB retries. (We just give some grace period to `pgx` to terminate.)